.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+-   id: pyproject-flake8
+    name: pyproject-flake8
+    description: '`pyproject-flake8` is a variant of `flake8` with support for configuration in the `pyproject.toml` file.'
+    entry: flake8
```

Comment:
Needs to be pflake8, otherwise it just calls the normal flake8 which will not check pyproject.toml